docs/prover/cli/options.md

```diff
@@ -379,34 +379,43 @@ Options affecting the type of verification run
 (--coverage_info)=
 ## `coverage_info`
 
-**Usage**
+**Parameters**
```

Comment:
Is "parameter" the right word for valid values? If we call an attribute that can be set with a flag on in a conf file "option" maybe these are "Option values"

Reply:
Changed to `Option values` everywhere